packages/components/organisms/f-status-banner/README.md

```diff
@@ -0,0 +1,131 @@
+<div align="center">
+
+# f-status-banner</h1>
```

Comment:
Not sure what's happened here but I think you need to use the `h1`?

Reply:
Can just remove the closing tag – I think this was a refactoring that was made. I'll update the generator too

Reply:
Fixed.